src/gourmet/timer.py

```diff
@@ -188,10 +189,17 @@ def note_changed_cb (self, entry):
     def init_player (self):
         self.player = Player()
 
-    def play_tune (self):
-        sound_file = self.sounds_and_files[cb.cb_get_active_text(self.soundComboBox)]
-        sound_file = os.path.join(gglobals.data_dir,'sound',sound_file)
-        self.player.play_file(sound_file)
+    def play_tune (self) -> None:
+        sound = self.sounds_and_files[cb.cb_get_active_text(self.soundComboBox)]
+        data = _get_data('gourmet', f'data/sound/{sound}')
+        assert data
+
+        # TODO!!! Delete the tempfile when we're done
+        # TODO: Figure out how to make GStreamer play raw bytes
+        fd, fname = _mkstemp('.ogg')
```

Comment:
Shouldn't this be `.opus`? (That's my last question)

Reply:
Opus is the encoding format and ogg is the container format, as [described here](https://en.wikipedia.org/wiki/Opus_(audio_format)#Containers). Although I'm seeing in the article that `.opus` is the recommended file extension. It doesn't really matter, especially being a temporary file, but I can certainly change it.